examples/LaserWakefield/include/simulation_defines/param/speciesInitialization.param

```diff
@@ -85,7 +85,7 @@ typedef mpl::vector<
 #if( PARAM_IONIZATION == 0 )
 
     CreateDensity<densityProfiles::Gaussian, startPosition::Random, PIC_Electrons>
-#   if( ENABLE_IONS == 1 )
+#   if( PARAM_IONS == 1 )
```

Comment:
I should remove this from the default params and add `speciesDefinition`the the `ThermalTest` example.